src/main/java/step3/domain/Lotto.java

```diff
@@ -1,36 +1,52 @@
 package step3.domain;
 
 import java.util.*;
+import java.util.stream.Collectors;
 
 public class Lotto {
 	private final Set<LottoNumber> numbers;
 
 	private static final int LOTTO_NUMBER_LENGTH = 6;
 	private static final String LOTTO_PHRASES = "로또 번호는 중복되지 않는 6개의 숫자로 이루어져야 합니다.";
 
+	public Lotto() {
+		this.numbers = issueAutoLotto();
+		validLotto();
```

Comment:
```suggestion
		validLotto();
		this.numbers = issueAutoLotto();
```
변수 할당을 하기전에 유효성 검사를 진행하는것과 그 후에 진행하는것은 어떤 차이가 있을까요 ?

Reply:
할당 전에 유효성 검사하도록 순서를 바꿔야 할 것 같긴했는데 그러려면 메소드 인자 수가 추가되어서 고민하던 부분이었어요!
메소드 인자 수도 줄이고 싶어서 저렇게 했는데 유효성 검사를 먼저 한다면 한 줄이라도 진행 과정을 덜어낼 수 있고, 불필요한 값 할당을 하지 않아도 될 것 같아요
유효성 검사를 먼저 하는 방법으로 수정하겠습니다~!